engines/griffon/cutscenes.cpp

```diff
@@ -182,6 +186,21 @@ void GriffonEngine::showLogos() {
 }
 
 void GriffonEngine::intro() {
+	#ifdef USE_TTS
```

Comment:
The indention here is off - the preprocessor line should be at the beginning of the line - and the guarded code itself is one level too deep.